test/file04.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { Theme } from '@xsite-ui/common.style';
+
+export const Container = styled.div<{ theme?: Theme }>`
+  display: grid;
+  gap: ${({ theme }) => theme?.space?.[5] || 20}px;
+  grid-template-columns: repeat(2, 1fr);
+
+  & [data-role='question'] {
```

Comment:
The attribute value selector would be more performant if it would have a prefix rather than an attribute selector. Using data-* will force a scan across all elements in the DOM.